src/board_filter.py

```diff
@@ -0,0 +1,78 @@
+"""
+This script is intended to be used with the related filter_finder.py. It filters out all the pixels corresponding to a single skateboard in a given controlled video of a skateboard under static conditions.
+
+TODO: make this script configurable to run from the commandline with no need to edit code, e.g. through `python board_filter -v "board.mp4" -c "wheels"`
+
+This script was created as part of a school project, and therefore license-free.
+"""
+
+import os.path
+import cv2
+import numpy as np
+
+__author__ = "Nick"
+
+
+#Increments by which values can be changed through keyboard controls
+sinc = 1
+binc = 5
+
+path = 'We have to manually enter the path to the video file here until we add support for commandline parameters.' #TODO: make path passable from the commandline
+
+#if path leads to valid file
+if os.path.isfile(path):
+	cap = cv2.VideoCapture(path)
+	
+	#try to read first frame
+	success, frame = cap.read()
+	
+	if(success):
+		#get dimensions of this video
+		height, width, _ = frame.shape
+
+		#lower and upper bounds for color detections, to be derived using keyboard controls. The default values given here are the averages of previous trials with Steve's skateboard.
+		lower_board = np.array([82,39,80])
```

Comment:
if these are constant, can we put it at the beginning of the file, so it doesn't clog up view